oca_dependencies.txt

```diff
@@ -0,0 +1,6 @@
+contract
+field-service
+geospatial
+odoo-addons https://github.com/ursais/odoo-addons.git 12.0
+stock-rma https://github.com/Eficent/stock-rma.git 12.0
+
```

Comment:
We don't need `oca_dependencies.txt` for `helpdesk_ticket_type_default_priority`. Please remove the file.